garak/resources/garak.core.yaml

```diff
@@ -13,6 +13,7 @@ run:
   eval_threshold: 0.5
   generations: 5
   probe_tags:
+  user_agent: "garak LLM vulnerability scanner, v{version} https://garak.ai"
```

Comment:
Per https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent, syntax should be:
```
User-Agent: <product> / <product-version> <comment>
```
```suggestion
  user_agent: "garak LLM vulnerability scanner/{version} https://garak.ai"
```

Reply:
Or:
```suggestion
  user_agent: "garak/v{version} (garak LLM vulnerability scanner, https://garak.ai)"
```